elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -318,6 +321,17 @@ export abstract class BaseTransaction {
 
 		errors.push(...assetErrors);
 
+		// Validate minimum remaining balance
+		const minRemainingBalanceError = verifyMinRemainingBalance(
+			this.id,
+			// Get updated account state
+			await store.account.get(this.senderId),
```

Comment:
sender already exist in above, so it's better to move the `store.account.set(sender.address, sender);` below this, or just reuse

Reply:
Sender don't have an updated balance. When we called `this.applyAsset(store)` The balance of the sender is changed in the store. The `sender` which exists in this context have old balance. So I have to fetch updated information from the store.

Reply:
ahhh ok that's true, it can changed in the previous step.
Maybe split the await statement above, and make bit more descriptive comment there?